src/main/resources/interface-specification.yml

```diff
@@ -349,43 +459,6 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
-  '/agreements/{agreementId}/contract':
```

Comment:
has this been deleted because moved to BFF?

Reply:
Yes.